closure/testing/phantomjs_harness.js

```diff
@@ -224,6 +236,7 @@ function onError(message, trace) {
  * @param {boolean} succeeded
  */
 function onCallback(succeeded) {
+  writeCoverage();
```

Comment:
Seems like Phantom can just report as coverage on this callback instead of creating another communication channel. If it is reported we can just write the results; we don't need another flag for phantom.